java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -859,6 +865,41 @@ public ColumnVector interleaveColumns() {
     return new ColumnVector(interleaveColumns(this.nativeHandle));
   }
 
+  /**
+   * Repeat the contents of this table count times.
```

Comment:
It's ambiguous whether the repetition is by row or by the entire table.  For example, a repeat count of 2 on 1, 2, 3 could be either 1, 1, 2, 2, 3, 3 or 1, 2, 3, 1, 2, 3 depending on whether it's interpreted to be a by row or by table repetition.  The docs says it repeats the table which implies the latter, but the unit test shows it's actually the former.